### DISCUSSION TITLE
Some toc look broken

### DISCUSSION THREAD

**Discussion by ChanYoung-dev** [mmistakes/minimal-mistakes#3679] [Q&A]
### What happened?

- Minimal Mistakes version: 4.24.0
- Ruby gem or remote theme version: 
- Jekyll version: 4.2.2
- Git repository URL: https://github.com/ChanYoung-dev/githubioLab
- Hosted on GitHub Pages (if yes provide URL to site): 
- Operating system: Mac

code
```md
---
permalink: /test/
title: "Basic"
toc: true
comments: true
---
# 0. Target
# 1. Jenkins
## 1.1 Configure
### 1.1.1 Source code
### 1.1.2 set build
### 1.1.3 Build
# 2. docker 
### Dockerfile
### dockerfile 
## ec2- docker 
# 3. nginx
### nginx 2
# 4. Docker-Compose
### docker-compose 
# 5. jenkins SSH 
## 5.1. 
## 5.2  ec2
## Jenkins job
```

<img width="969" alt="image" src="https://user-images.githubusercontent.com/46098949/172053871-6363c574-8a2d-42a9-aa02-4741d6f9e681.png">

### Expected behavior

I installed Minimal mistakes and posted one post without touching anything. But from the 11th Header, it looks broken. What's the reason?

**Comment by iBug** [comment]
You should not use "jumping" heading levels. A heading level N should only be followed by a heading of level N+1 (or smaller).

**Reply by ChanYoung-dev** [reply]
Thanks !
